### PR TITLE
feat: Support untrusted workspace

### LINKED ISSUE #17: Support untrusted workspace
We can support the new VS Code security feature - untrusted workspace directly,
since this extension do not depend on any part of the untrusted code base.

There since no document page about this. Currently it's only mentioned in VS Code release notes:
<https://code.visualstudio.com/updates/v1_56>

> You can declare your extension to provide complete or partial or no support in untrusted workspaces using the `untrustedWorkspaces` capability in `package.json`.

```json
"capabilities": {
  "untrustedWorkspaces": {
    "supported": true
  }
}
```

### PR DESCRIPTION
**Pull request by AnnAngela**
Resolve #17

Support untrusted workspace. When running in a untrusted workspace, these behaviors will happen:
* The workspace-defined vscode configuration will not be used;
* The workspace-defined git configuration which came from `./.git` will not be used, the `git` command will be executed in an empty temp folder instead.

Currently I got a problem when debugging, I cannot activate the extension in restricted mode, even reinstalling it via local `.vsix` package. [The document](https://code.visualstudio.com/api/extension-guides/workspace-trust) doesn't say how to do...

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -12,7 +12,7 @@
         "name": "Weida Hong",
         "url": "https://blog.bitisle.net"
     },
-    "version": "0.6.2",
+    "version": "0.7.0-alpha",
```
Comment (AnnAngela) [inline package.json:15]: This change is used to indicated that the local installed version is debugging one.

**Comment by wdhongtw** [comment]
> I cannot activate the extension in restricted mode

I known what happened. It's something related to extension packaging.
I'll fix it these days. :D

**Comment by wdhongtw** [comment]
I have done the testing and it works well.  Thanks @AnnAngela  :D

**Comment by AnnAngela** [comment]
> I have done the testing and it works well.  Thanks @AnnAngela  :D

THX! 

BTW, if convenient, can you publish a pre-release version?

**Pull request merged**